scripts/zones/Den_of_Rancor/mobs/Hakutaku.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- Area: Den of Rancor
+--  Mob: Hakutaku
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.GIL_MIN, 1600)
+    mob:setMobMod(xi.mobMod.GIL_MAX, 3500)
+    mob:setMod(xi.mod.TRIPLE_ATTACK, 45)
```

Comment:
Got a source for this?

I could only find:
https://ffxiclopedia.fandom.com/wiki/Hakutaku
https://ffxiclopedia.fandom.com/wiki/Talk:Hakutaku#Possible_Triple_Attack?

Which mentions: `20-30% Triple Attack rate`

Reply:
People debating it on the dtalk page with some having NEVER seen it and others saying low 4rate but then mobs main page said "high rate" and pulls 20-30 from its ass. _No, that's not a trustworthy source._ #shitWikiSays strikes again. And nothing says 45. Even if it does have it, I'd expect it provided via its subjob not a raw modifier.